package.json

```diff
@@ -1,24 +1,107 @@
 {
-  "name": "go-test",
-  "version": "1.0.0",
-  "description": "",
+  "name": "go-template",
+  "version": "0.0.1",
+  "description": "Template for generating Go module based on async api",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"no test specified yet\"",
-    "lint": "echo \"no lint configured yet\"",
-    "generate:assets": "echo \"no assets to generate yet\"",
-    "develop": "ag test/asyncapi.yaml ./ -o test/output --force-write --watch-template"
+    "test": "npm run testTemplate && npm run testHooks && npm run testFilters",
+    "testTemplate": "rimraf test/temp && jest --coverage -t 'templateGenerationResult' --passWithNoTests",
+    "testHooks": "rimraf test/temp && jest --coverage -t 'generateExtraFormats' --passWithNoTests",
+    "testFilters": "rimraf test/temp && jest --coverage -t 'generateMermaidDiagram' --passWithNoTests",
```

Comment:
I don't think you have tests here in the project yet, so just keep `test` script that echos there are no tests. And as a followup create an issue to add snapshot tests that are a kind of integration tests checking if generated output is what you expected. I can later introduce you to it, not a problem at all

Reply:
ok done, make sense